src/server/js/resources/users/index.js

```diff
@@ -1,16 +1,24 @@
 import express from 'express';
+import expressJwt from 'express-jwt';
 
 import UserModel from './model';
+import passport from '../../passport';
+import HouseModel from '../houses/model';
 
 const router = new express.Router();
 
-router.get('/', (req, res, next) => {
-    return UserModel.find({}, (err, users) => {
+const auth = expressJwt({
+    secret: process.env.JWT_SECRET,
+    userProperty: 'payload',
+});
+
+router.get('/', auth, (req, res, next) => {
+    return UserModel.findById(req.payload.id, (err, user) => {
```

Comment:
now looking back, maybe it's weird to be getting just one user (not in an array) from the list route?

Reply:
yeah, just wrap the single user in an array for a response